src/hooks/useReactions.tsx

```diff
@@ -0,0 +1,11 @@
+import db, { Message, MessageReaction } from "../model/db";
+import { useLiveQuery } from "dexie-react-hooks";
+
+export function useReactions(message: Message): MessageReaction[] | undefined {
+  return useLiveQuery(async () => {
+    return await db.reactions
```

Comment:
do you want to wrap this in a try/catch or do any error handling here?

Reply:
I'm not sure what would throw here unless there's something weird going on with dexie. Since this is just trying to be a conceptual demo and not an end user app, I'm not as worried about handling upstream dep errors.